relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -629,6 +680,57 @@ mod tests {
                     "transaction.status": "ok",
                 },
             },
+            Bucket {
+                timestamp: UnixTimestamp(1619420400),
+                width: 0,
+                name: "d:transactions/measurements.score.lcp@ratio",
+                value: Distribution(
+                    [
+                        0.0,
+                    ],
+                ),
+                tags: {
+                    "browser.name": "Chrome",
+                    "environment": "fake_environment",
+                    "release": "1.2.3",
+                    "transaction": "gEt /api/:version/users/",
+                    "transaction.op": "mYOp",
```

Comment:
Without the tag filter, this would include platform, dist, etc.

Reply:
@edwardgou-sentry @k-fish apart from the performance score UI, could there be a more generic measurements-related part of the UI that breaks if these metrics come with only a subset of tags?

Reply:
As far as I know, Dashboards is the only other place users would be able to write a query for Performance Scores using metrics. But I'm pretty sure that if a metrics query breaks due to missing tags, it will fall back to the indexed dataset instead.